assets/stylesheets/modules/speakers.scss

```diff
@@ -90,7 +90,11 @@
     .speaker__name {
       color: $pink;
     }
-    .index__speaker__image {
+    .speaker__bio__image {
+      border: 6px solid $pink;
+    }
+    .speaker__bio__info {
+      opacity: 1;
       border: 6px solid $pink;
```

Comment:
With a pink border and ping background nope we don't need it. The opacity wont mess with the border color now.